src/utils/ctl/cart_ctl.c

```diff
@@ -263,6 +267,8 @@ print_usage_msg(const char *msg)
 	printf("\tdon't perform 'wait for ranks' sync\n");
 	printf("-m 'log_message'\n");
 	printf("\tSpecify log message to be sent to remote server\n");
+	printf("--use-daos-agent-env\n");
+        printf("\tSet OFI and CRT_* vars through daos_agent\n");
```

Comment:
(style)  code indent should use tabs where possible

Reply:
style issue needs to be fixed

Reply:
43fe7d9e15e15bf4a8cb6c831dd916cadb84f7bd